### DISCUSSION TITLE
Email stuck in an unverified loop

### DISCUSSION THREAD

**Discussion by nitlix** [vercel/community#4312] [Account Issue] [answered]
### Question

I'm currently trying to change my email on my main vercel account to another one, but when I try to do so, it says: "Email not available".

When I try to log in with it, it asks for a phone number, however, my phone number is linked to my main vercel account. What should I do in this case?

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble with your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. If so, please follow those steps.

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
